tests/schema/models.py

```diff
@@ -124,6 +124,13 @@ class Meta:
         db_table = "schema_note"
 
 
+class NoteInfoIndexed(models.Model):
```

Comment:
Is there a reason you use a regular model here but an inner model in the other test?